### ISSUE TITLE
Shortcut keys conflict

### ISSUE THREAD

**Issue by PtJade-Ceramic** [musescore/MuseScore#27323]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

The shortcut key for **Tools** (**J**) is taken up by that for **Add** (**J**).
1. Press **Alt**, **J**.
2. Always only **Add** can be expanded.

### Supporting files, videos and screenshots

In the menu bar, the shortcut key for both **Add** and **Tools** are the same.

<img width="331" alt="Image" src="https://github.com/user-attachments/assets/6faa1a84-ea0e-4af9-b61b-d0942c060459" />

### What is the latest version of MuseScore Studio where this issue is present?

OS: Windows 11 Version 2009 or later, Arch.: x86_64, MuseScore Studio version (64-bit): 4.5.0-250721848, revision: github-musescore-musescore-7c55023

### Regression

I was unable to check

### Operating system

版本	Windows 11 家庭版 版本号	24H2 操作系统版本	26100.3476 体验	Windows 功能体验包 1000.26100.54.0

### Additional context

[Menu bar](https://handbook.musescore.org/navigation/the-user-interface#menu-bar)

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by PtJade-Ceramic** [comment]
> These shortcuts or accelerator keys or menmonics are something the translators take care of, they are not part of the code itself, so can't get fixed there. This issue here exists only in the Chinese (China) translation, not in Chinese (Hongkong) nor in Chinese (Taiwan)
> 
> I've now changed the translations to use A for Add and T for Tools, just like the 2 other Chinese variants do (and like it's been up to 10 months ago). Update your translations in some 15 minutes and restart MuseScore to see this

I have done what you suggest but the key **T** is taken up by **View** now.

<img width="355" alt="Image" src="https://github.com/user-attachments/assets/8ddd8b93-0877-46fd-85c0-1126553827cf" />

**Comment by Jojo-Schmitz** [comment]
Getting those nmemonics not to conflict with one another, i.e. having them being unique in their menu level, is a pretty tough task for the translators

**Comment by bkunda** [comment]
If this is not something that can be fixed in MSS's code, then it should probably be closed for now and handled separately (If indeed it is further resolvable).

**Comment by Jojo-Schmitz** [comment]
This indeed is something that probably should not have been reported on GitHub.
However, it is fixed now anyway, via fixing a translation on Transifex

**Comment by PtJade-Ceramic** [comment]
> This is something that should not have been reported on GitHub. However, it is fixed now anyway, via fixing a translation on Transifex.

There are many other similar conflicts even in the latest version (specific for Simplified Chinese (zh CN)), as is illustrated hereunder:
<img width="194" alt="Image" src="https://github.com/user-attachments/assets/096576d8-a634-472b-ab99-ff1097cbe26b" />
and can you tell me how to modify mnemonic settings, which I can't find now on [MuseScore | Transifex](https://app.transifex.com/musescore/musescore/translate/#zh_CN/musescore/431867054), please?

**Comment by Jojo-Schmitz** [comment]
See https://app.transifex.com/musescore/musescore/translate/#zh_CN/$/570653196?q=translation_text%3A(%26M)
and https://app.transifex.com/musescore/musescore/translate/#zh_CN/$/570653196?q=translation_text%3A(%26L)
